app/services/analytics_events.rb

```diff
@@ -2773,37 +2773,6 @@ def idv_in_person_proofing_address_visited(
     )
   end
 
-  # @param ["hybrid","standard"] flow_path Document capture user flow
-  # @param [String] step
-  # @param [String] analytics_id
-  # @param [Boolean] success Whether form validation was successful
-  # @param [Hash] errors Errors resulting from form validation
-  # @param [Hash] error_details Details for errors that occurred in unsuccessful submission
-  # @param [Boolean] same_address_as_id
-  # User clicked cancel on update state id page
-  def idv_in_person_proofing_cancel_update_state_id(
-    success:,
-    errors:,
-    flow_path: nil,
-    step: nil,
-    analytics_id: nil,
-    error_details: nil,
-    same_address_as_id: nil,
-    **extra
-  )
-    track_event(
-      'IdV: in person proofing cancel_update_state_id submitted',
```

Comment:
I wonder if this was missed functionally that needed to be added to the controller.

Reply:
Good question! I double checked, and the state id controller `show` and `update` methods both do have analytics being tracked, which seem more up to date. 

So I think maybe these analytics were replaced with `idv_in_person_proofing_state_id_visited` and `idv_in_person_proofing_state_id_submitted` and these removed ones are old.


<details>
<summary><b>Show:</b></summary>
<br>

https://github.com/18F/identity-idp/blob/d220668a71f71b77f16a07f5399db430fbe68576/app/controllers/idv/in_person/state_id_controller.rb#L14

https://github.com/18F/identity-idp/blob/b1d8e1893974a9492ad1ff56e7a5b1da8eddc9f9/app/services/analytics_events.rb#L2934-L2953

</details>

<details>
<summary><b>Update:</b></summary>
<br>

https://github.com/18F/identity-idp/blob/d220668a71f71b77f16a07f5399db430fbe68576/app/controllers/idv/in_person/state_id_controller.rb#L33-L35

https://github.com/18F/identity-idp/blob/b1d8e1893974a9492ad1ff56e7a5b1da8eddc9f9/app/services/analytics_events.rb#L2896-L2925

</details>

Reply:
It does seem like each of the update flows were updated to exclude the redo and cancel update events with the exception of ssn_redo but I wonder if that was a miss... I think we can continue without those events. It might be good to double check with Sumi.

Reply:
Good point that these were also removed from the update flows in the other steps. I agree we can go on without these analytics because they appear deprecated.